core/store/src/trie/trie_recording.rs

```diff
@@ -29,6 +29,14 @@ impl TrieRecorder {
         }
     }
 
+    /// Records value without increasing the size.
```

Comment:
nit. without increasing the recorded witness size? "the size" is ambiguous.

Reply:
good point, done in dc9295f56